lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -8,13 +8,15 @@
 from django.core.urlresolvers import reverse
 from django.test.client import RequestFactory
 from django.test.utils import override_settings
+from edxmako.shortcuts import render_to_response  # pylint: disable=import-error
```

Comment:
I don't think you need this disable; it can erroneously happen on devstack but shouldn't happen on Jenkins. To fix devstack, you need to run `paver run_pylint` prior to running `paver run_quality`. The first task properly pulls in all the imports, but takes awhile.